docs/docs-next/docs/guides/automation.md

```diff
@@ -21,117 +21,59 @@ Automation is key to building reliable, efficient data pipelines. This guide cov
+For examples of how to create schedules, see the [How-To Use Schedules](/guides/automation/schedules) guide.
 
-# Create a schedule that runs the job daily at midnight
-daily_schedule = ScheduleDefinition(
-    job=daily_refresh_job,
-    cron_schedule="0 0 * * *"  # Runs at midnight daily
-)
-```
-
-View more detailed examples of schedules in the [How-To Use Schedules](/guides/automation/schedules) 
-and read more about how Schedules work in [About Schedules](/concepts/schedules).
+For more information about how Schedules work, see the [About Schedules](/concepts/schedules) concept page.
 
 ## Sensors
 
 Sensors allow you to trigger runs based on events or conditions, like a new file arriving or an external system status change.
 
-A sensor requires that you define a function that will 
+Like schedules, sensors operate on a selection of assets, known as [Jobs](/concepts/ops-jobs) and can either start a pipeline 
+through a Run or log a reason for not starting a pipeline using a SkipReason.
+
+However, unlike schedules, sensors are triggered by events that you define.
+You must provide a function that the sensor will use to determine if it should trigger a run.
 
 ### When to use Sensors
 
 - You need event-driven automation
 - You want to react to changes in external systems
 
-### Basic Sensor Example 
-
-```python
-from dagster import RunRequest, SensorDefinition, sensor
-
-@asset
-def my_asset():
-    ...
-
-my_job = define_asset_job("my_job", selection=[my_asset])
+For more examples of how to create sensors, see the [How-To Use Sensors](/guides/automation/sensors) guide.
 
-def check_for_new_files() -> List[str]:
-    return ["file1", "file2"]
+For more information about how Sensors work, see the [About Sensors](/concepts/sensors) concept page.
 
-@sensor(job=my_job)
-def new_file_sensor():
-    new_files = check_for_new_files()
-    if new_files:
-        yield RunRequest(run_key=f"filename")
-
-```
-
-## 3. Declarative Automation
-
-Declarative Automation allows you to automatically materialize assets when specified criteria are met, without needing to define explicit jobs.
-
-### When to use Declarative Automation
-
-- You're working primarily with assets
-- You want a simpler, more declarative approach to automation
-
-### Basic Declarative Automation Example
-
-```python
-from dagster import asset, AutoMaterializePolicy, AutoMaterializeRule
-
-@asset(
-    auto_materialize_policy=AutoMaterializePolicy(
-        rules=[
-            # Materialize if upstream assets have changed
-            AutoMaterializeRule.materialize_on_parent_updated(),
-            # Materialize daily at 2 AM
-            AutoMaterializeRule.materialize_on_cron("0 2 * * *"),
-        ]
-    )
-)
-def my_asset():
-    # Asset computation logic here
-    pass
-```
-
-## 4. Asset Sensors
+## Asset Sensors
 
 Asset Sensors trigger jobs when specified assets are materialized, allowing you to create dependencies between jobs or code locations.
```

Comment:
The `Sensors` in `Asset Sensors` doesn't need to be capitalized

Reply:
Asset Sensor is a Dagster Concept: https://docs.dagster.io/_apidocs/schedules-sensors#dagster.AssetSensorDefinition
there's a decorator called `@asset_sensor` too.